arrival-notice/src/main/java/org/dcsa/conformance/standards/an/checks/ANChecks.java

```diff
@@ -134,7 +129,20 @@ public static JsonContentCheck validateDataSetFields(
   public static final JsonContentCheck VALIDATE_NON_EMPTY_RESPONSE =
       JsonAttribute.customValidator(
           "Every response received during a conformance test must not be empty",
-          body -> ConformanceCheckResult.simple(body.isEmpty() ? Set.of("The response body must not be empty") : Set.of()));
+          body ->
+              ConformanceCheckResult.simple(
+                  (body.isEmpty() || body.get("arrivalNotices").isEmpty())
```

Comment:
I think this is enough


```suggestion
                  body.get("arrivalNotices").isEmpty()
```